assets/src/components/cd/services/service/ServiceDetailsSidecar.tsx

```diff
@@ -103,6 +139,24 @@ export function ServiceDetailsSidecar({
             {helm.version}
           </SidecarItem>
         )}
+        {repository && (
+          <SidecarItem heading="Git repository">
+            <div
```

Comment:
Is there not some reusable component for this already?  If there isn't we probably should make one and put it in the utils library, can't imagine we won't need more icon-ed strings in the future.